packages/core/docs/guide/error-handling.md

```diff
@@ -36,26 +37,23 @@ export interface UseCartErrors {
 }
 ```
 
-
 :::details Where does the error come from?
 
-Inside each async method we are clearing the current error before integration's method call and setting it in catch block.
+Inside each async method, we are clearing the current error before an integration's method call and setting it in the `catch` block.
+
```

Comment:
I would delete the example below as it's a part of core, while core is something that end-user doesn't have to be aware of. For him, the knowledge about what he can gets from error fields is enough. I know it was merged as it is previously, but now when I'm reading this it sounds like we are explaining our implementation instead of how this API works.

I suggest:
--- START --- 
Inside each async method, we are catching errors when they occur and save them to the reactive property called `errors` under the key corresponding to the triggered method. 

Example:
```js
const { addToCart, errors } = useCart()

addToCart({ product: null }) // triggers an error
errors.addToCart // here you have error raised by addToCart function 
```
--- END --- 

and ofc delete the current example

Reply:
@andrzejewsky thank you for the suggestion! Done 👍🏻